### ISSUE TITLE
Chapter 13 - WebBitmap is blank

### ISSUE THREAD

**Issue by rclancy** [xamarin/xamarin-forms-book-samples#13]
Both the code and XAML versions of WebBitmap produce a blank screen on my Android emulator (VS2015, Hyper-V, 5" KitKat Emulator, Windows10). The same Universal Windows and Windows 8.1 projects do show an image. It is only the Android emulator that has the problem, though I haven't tried iOS or Windows Phone. 
The url also shows an image if I navigate to it in a browser. Is this a security issue? Could it be a firewall issue? This is the image: http://developer.xamarin.com/demo/IMG_1415.JPG

The ImageBrowser Android project fails also, probably for the same reason. No images.
Similarly the BitmapStreams project displays the resource stream but not the WebRequest stream.

Same with Chapter 16 WebViewDemo. It works perfectly on UWP or Windows but displays nothing on the Android emulator running in Hyper-V.

This is a Hyper-V emulator IP / router problem. The Android emulator does not have a valid IP address. I'm working with a router technician now... This can be closed.
(The problem was IPS detection. When I turned it off, everything worked)

**Comment by charlespetzold** [comment]
I can’t reproduce your problem. Screenshot is attached.

Charles

**Comment by rclancy** [comment]
It was a firewall IPS filter on my SonicWall router. IPS had been enabled for years without issue, but the Hyper-V emulator requests were blocked by it. I turned off the feature on the firewall and all is now working. Interestingly, the IPS setting was available on several zones on the router, and enabled on WAN and LAN. When I disabled it on WAN, half of your samples which had the problem suddenly worked (ImageBrowser for example), but it wasn’t till I disabled it on LAN that the other half started returning images (WebBitmap for example). The problem did teach me about the emulator’s MAC address and IP leases, which was useful
